### ISSUE TITLE
Documentation Update

### ISSUE THREAD

**Issue by dan64** [jantic/DeOldify#153]
I like DeOldify, It has already a great potential. But after being able to install it successfully I suggest you to improve the documentation inside the README.md. Find below my proposals:

1) DeOldify run also in Windows 10, not only but I discovered the the GPU memory is managed much better in Windows 10 than in Linux (maybe because any memory leaks are discovered much faster by the Games developers).
2)  Using Anaconda (as you suggested) the command "source activate deoldify" should be changed in "conda activate deoldify"
3) You should enforce (both in the readme and in the creation scripts) the installation from the pytorch channel of the following packages:  pytorch=1.1.0,  torchvision=0.3.0 which are able to run smoothly in DeOldify (unfortunately pytorch=1.2.0 is not working with  DeOldify).

Thanks again for sharing this useful project. I'm new to Deep Learning but after reading this project I found the subject interesting.

**Comment by dan64** [comment]
In effect doing more testing I discovered that in Windows the render_factor is not properly managed. The software will produce the same image quality even using a render_factor=256. Not only but the final image in windows is worse, as it is possible to see looking to the images bellow:

WINDOWS 10
[https://i.ibb.co/nwPrzHw/windows-96.jpg](url)
LINUX CR=25
[https://i.ibb.co/BV5fL4L/linux-cr-25-96.jpg](url)
LINUX CR=38
[https://i.ibb.co/3Nm3cgR/linux-cr-38-96.jpg](url)

**Comment by dan64** [comment]
I have good news. I reinstalled the DeOldify env and reinstalled all the necessary packeges. Now I have almost the same quality and speed obtained in Linux. But the GPU memory usage in Windows is lower respect to Linux I was able to encode a short HD video (1090x1080) with a render_factor of 55 consuming about 5GB of memory in Linux I can use a max render_factor of 38, then I get out of memory. Moreover I cannot see any "visible" difference between RF 38 and 55. The situation change if I use ImageColorizer, in this case I can use max render_factor=44. I attached the images obtained with ImageColorizer.

WINDOWS RF=21
![century20th_w10_RF=21](https://user-images.githubusercontent.com/5787502/63881812-eb535b80-c9d0-11e9-8198-82bbc3a16c2c.jpg)

WINDOWS RF=38
![century20th_w10_RF=38](https://user-images.githubusercontent.com/5787502/63881823-f0b0a600-c9d0-11e9-9c73-42d1ea2a0764.jpg)

WINDOWS RF=44
![century20th_w10_RF=44](https://user-images.githubusercontent.com/5787502/63881832-f4dcc380-c9d0-11e9-9816-5f38b22dcbef.jpg)

This is my env in Windows 10:
[conda-list.txt](https://github.com/jantic/DeOldify/files/3552269/conda-list.txt)

**Comment by dan64** [comment]
by mistake I pushed the button "close and comment" :-)

**Comment by jantic** [comment]
Ok I've addressed #2 and #3.  It's actually PyTorch 1.0.1 that we need (I've tested), and the TorchVision requirement is taken care of automatically by the install of PyTorch it appears (it installs 0.2.1).  FastAI's install should have taken care of the PyTorch requirement correctly but it just grabs the latest version.  So now it's patched over in our own install.

Something to point out- FastAI 1.0.51 is also specifically needed.  If you go higher  images will be rendered with weird grid artifacts.  Oye.  

In summary- for all those reading: Please please please just use the conda install.  It'll save you a lot of headache.
